### PR TITLE
[DSM] PEPPER-271 Singular field changes

### PR DESCRIPTION
**Pull request by pegahtah**
## Context

PEPPER-271 liquibase created with queries to update field_settings table with inserting the new fields and updating order numbers for other fields

## Checklist

- [ ] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [x] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="PEPPER-271_GROUP_CLINICAL_NOTES" author="pegah">
+        <sql>
+            SELECT @realm:= ddp_instance_id FROM ddp_instance WHERE instance_name='singular';
+            SELECT @orderNumber := order_number FROM dev_dsm_db.field_settings where ddp_instance_id = 32 and column_display = 'Protein-losing enteropathy' and
+            field_type = 'GROUP_CLINICAL_NOTES' ;
+
+            update field_settings set order_number = order_number + 2
+            where field_settings_id in ( select field_settings_id from (select * from field_settings) as tmp
+            where field_type = 'GROUP_CLINICAL_NOTES' and order_number >= @orderNumber) and field_settings_id &lt;&gt; 0;
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml]: should we change `>` to `&lt;` as well?

Comment (pegahtah) [reply pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml]: ughh yes thank you

File: pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+            INSERT INTO field_settings (ddp_instance_id, field_type, column_name, column_display, display_type, order_number,
+            possible_values )
+            SELECT ddp_instance_id , 'GROUP_CLINICAL_NOTES', 'PRIOR_LIVER_TRANSPLANT' ,'Prior listing for liver transplant',
+            'RADIO', @orderNumber,  '[{"value":"YES","name":"Yes"},{"value":"NO","name":"No"},{"value":"UNKNOWN","name":"Unknown"}]'
+            from ddp_instance where instance_name = 'Singular';
```
Comment (SimoneMaiwald) [inline pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml]: instance_name here and in L27 `Singular` up in L9 `singular`
does it matter?

File: pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml (hunk 1 above)
Comment (pegahtah) [reply pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml]: (but it worked in dev when I ran it) 🤔

File: pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml (hunk 2 above)
Comment (pegahtah) [reply pepper-apis/dsm-core/src/main/resources/liquibase/singular/PEPPER-271_clinicalNotesChanges.xml]: No I don't think SQL cares for VARCHAR but I will update it

**Review by ngigliotosBroad — DISMISSED** [review]

**Review by SimoneMaiwald — APPROVED** [review]

**Review by ngigliotosBroad — APPROVED** [review]

**Pull request merged**